docs/install_gpu.rst

```diff
@@ -92,3 +92,35 @@ Assuming ``pytest`` is installed (otherwise, just run ``pip install pytest``), y
 
 In contrast to the C interface tests, these check for correctness, so a successful test run signifies that the library is working correctly.
 Note that you can specify other framework (``pycuda``, ``torch``, or ``numba``) for testing using the ``--framework`` argument.
+
+
+Matlab interface
+----------------
+
+.. _install-matlab-gpu:
+
+In addition to the Python interface, cuFINUFFT also comes with a Matlab interface. To install this, you first build the shared library.
+For example, assuming in the root directory of finufft, then run
+
+.. code-block:: bash
+
+    cmake -S . -B build -D FINUFFT_USE_CUDA=ON -D FINUFFT_STATIC_LINKING=OFF -D CMAKE_VERBOSE_MAKEFILE:BOOL=ON -D FINUFFT_CUDA_ARCHITECTURES="60;70;80;90"
```

Comment:
we don't reallly need to specify the architectures as it should use native by default.

Reply:
Yes, we don't have to. But I find it's a bit annoying compiled on my FI cluster workstation with P4000 card, later testing on v100, a100 and v100 recompiling every time, at least for cluster users compiling with different arches save some time.

Reply:
At the time of the PR draft, FINUFFT_CUDA_ARCHITECTURE was used for compiling with different compute capabilities and there was a duplicate CMAKE_CUDA_ARCHITECTURE flag causing confusion. Later, @DiamonDinoia cleared the ambitious and now only CMAKE_CUDA_ARCHITECTURE is used. I'm making the corresponding tweaks for the docs now.